lspc.py

```diff
@@ -22,20 +22,23 @@
 def edit_uri_select(uri, positions):
     filename = utils.uri_to_file(uri)
     if filename:
-        return 'edit {}; {}'.format(filename, libkak.select(positions))
+        return 'edit {}; {}'.format(filename, libkak.select((positions[1],positions[0])))
```

Comment:
Thanks, can you revert this reordering too?

Reply:
oh yeah, of course, totally missed that